runners/genlayer-py-std/src/genlayer/nondet/__init__.py

```
@@ -105,15 +109,17 @@ def exec_prompt(
 		else:
 			images.append(im)
 
+	format = config.get('response_format', 'text')
```

Ruff (0.15.10)

[error] Variable `format` is shadowing a Python builtin

(A001)